cli/schemas/config-file.v1.json

```diff
@@ -547,7 +547,7 @@
     "name": {
       "type": "string",
       "description": "The name of this JSR package. Must be scoped",
-      "examples": ["@luca/flag"]
+      "examples": ["@scope/name"]
```

Comment:
Actually, I find anything showing up here kind of annoying. Let's maybe remove this and make it a regex instead.